tests/ui_tests/app/configuration.py

```diff
@@ -26,23 +28,13 @@ def __init__(self, driver, url=None):
         # 30 seconds timeout here is caused by possible long load of config page
         self._wait_element_present(ConfigurationLocators.load_marker, 30)
 
+    @allure.step('Check that we can edit field or not')
     def assert_field_editable(self, field, editable=True):
-        """Check that we can edit specific field or not
-        :param field:
-        :param editable:
-        :return:
-        """
         field_editable = self.editable_element(field)
         assert field_editable == editable
 
+    @allure.step('Check that field content equal expected value: {expected_value}')
```

Comment:
```suggestion
    @allure.step('Check that field content is equal to expected value: {expected_value}')
```

Information about field we are investigating we be also usefull